mix.exs

```diff
@@ -22,9 +22,9 @@ defmodule OpentelemetryPlug.MixProject do
   defp deps do
     [
       {:hackney, "~> 1.0", only: :test, runtime: false},
-      {:opentelemetry_api, "~> 0.3"},
-      {:opentelemetry, "~> 0.3"},
-      {:plug, "~> 1.10.1"},
+      {:opentelemetry_api, "~> 1.0.0-rc"},
+      {:opentelemetry, "~> 1.0.0-rc", only: :test},
+      {:plug, "~> 1.11.1"},
```

Comment:
We should attempt to keep this as relaxed as we possibly can. Did something change in 1.11 that this implementation relies on?

Reply:
The [plug router telemetry events were added after 1.10.1](https://github.com/elixir-plug/plug/commit/b753a2ffc24cdd41bab0cb576a5827954f8f5d0a#diff-8abb6a1c8f0384efb997eebfa1f2a8b3f91e339b95ba79de9a50132b3ddddd3c), thanks to @tsloughter.

Reply:
Yeah, that commit was released in 1.10.1

https://github.com/elixir-plug/plug/compare/v1.10.0...v1.10.1

Reply:
Relaxed in 89d05d2